MastodonSDK/Sources/MastodonUI/View/Content/NotificationView+ViewModel.swift

```diff
@@ -125,6 +125,76 @@ extension NotificationView.ViewModel {
                 }
             }
             .store(in: &disposeBag)
+
+        Publishers.CombineLatest4(
+            $authorName,
+            $authorUsername,
+            $notificationIndicatorText,
+            formattedTimestamp
+        )
+        .sink { name, username, type, timestamp in
+            notificationView.accessibilityLabel = [
+                "\(name?.string ?? "") \(type?.string ?? "")",
+                username.map { "@\($0)" } ?? "",
+                timestamp
+            ].joined(separator: ", ")
+            if !notificationView.statusView.isHidden {
+                notificationView.accessibilityLabel! += ", " + (notificationView.statusView.accessibilityLabel ?? "")
+            }
+            if !notificationView.quoteStatusViewContainerView.isHidden {
+                notificationView.accessibilityLabel! += ", " + (notificationView.quoteStatusView.accessibilityLabel ?? "")
+            }
+        }
+        .store(in: &disposeBag)
+
+        Publishers.CombineLatest(
+            $authorAvatarImage,
+            $type
+        )
+        .sink { avatarImage, type in
+            var actions = [UIAccessibilityCustomAction]()
+
+            // these notifications can be directly actioned to view the profile
+            if type != .follow, type != .followRequest {
```

Comment:
Can't we also append this action to a `follow` event? Without Voice Over I'm able to open up the profile from a follow-notification, with Voice Over enabled I'm not anymore.

Reply:
You should be able to use the default “activate” action to open the profile for these types of notifications, whereas for other notifications that action will open the corresponding post instead.

Reply:
That’s very weird, on iOS 16.2 the action works fine. Maybe it has something to do with focus or something? I don’t have a person following me at the top of my notification feed so I guess perhaps there could be an issue with the first one or something? If you’re able to figure out in more detail what’s going on that would be awesome :)

PS: You can turn on the “Caption Panel” (at the bottom of VoiceOver settings) to get a small panel at the bottom of your screen that lets you visually read what VoiceOver is saying, which can be useful for debugging accessibility issues sometimes)

Reply:
@j-f1 I managed to trigger the default open action on the item now, looks like I released my finger to quickly. That being said, if you resolve the conflicts we could merge this PR.